internal/provider/project_resource.go

```diff
@@ -134,10 +134,13 @@ func (r *ProjectResource) Create(ctx context.Context, req resource.CreateRequest
 		return
 	}
 
-	plan, diags = DTProjectToTFProject(ctx, respProject)
+	state, diags = DTProjectToTFProject(ctx, respProject)
 	resp.Diagnostics.Append(diags...)
 
-	resp.Diagnostics.Append(resp.State.Set(ctx, &plan)...)
+	// API does not return parent ID when updating, so we assume it was set as requested
```

Comment:
The API does not return the parent also in case of create, so applied the same solution here as the one in update.